docs-src/references/configuration.md

```diff
@@ -387,3 +387,123 @@ publicClient:
 ```
 
 Use `dns:///` prefix to enable round-robin between IP address for DNS name.
+
+## archival
+
+_Optional_
+
+Archival is an optional configuration needed to set up the [Archival store](/concepts/what-is-archival).
+It can be enabled on the `history` and `visibility`.
```

Comment:
history, visibility what? :) 
i think you just configure archival, not sure there is option to select if you only want event history or visibility data stored